scripts/serverless.js

```diff
@@ -173,10 +173,10 @@ const processSpanPromise = (async () => {
         'NEW_VARIABLES_RESOLVER',
         `"${humanizedPropertyPath}" is not accessible ` +
           '(configured behind variables which cannot be resolved at this stage).\n' +
-          'Starting with next major release, ' +
-          'this will be communicated with a thrown error.\n' +
-          'Set "variablesResolutionMode: 20210326" in your service config, ' +
-          'to adapt to this behavior now',
+          'In the next major version ' +
+          'this will result in an error.\n' +
+          'Set "variablesResolutionMode: 20210326" in your service config ' +
+          'to opt in this behavior now',
```

Comment:
I think it should say "opt in to this behavior" instead of current version - could you confirm that?

Reply:
👍  fixed, thanks!